packages/design-system/src/styles/components/_Button.scss

```diff
@@ -182,21 +181,26 @@
 %transparent-inverse-button {
   border-color: transparent;
   background-color: transparent;
-  // Default inverse focus styles are overridden above
-  @if $ds-include-focus-styles {
-    &:focus,
-    &.ds-c-button--focus {
-      border-color: transparent;
-    }
-  }
 
   &:disabled,
-  &:hover,
   &:active,
   &.ds-c-button--disabled,
-  &.ds-c-button--hover,
   &.ds-c-button--active {
     border-color: transparent;
+    background-color: transparent;
+  }
+  &:hover,
+  &.ds-c-button--hover {
+    color: rgba($color-base-inverse, 0.8);
+  }
+  // Default inverse focus styles are overridden above
+  @if $ds-include-focus-styles {
```

Comment:
Moved focus styles below to make sure order was correct

Reply:
Oh is this to address DK's commnt here? https://github.com/CMSgov/design-system/pull/946#pullrequestreview-596579401